source/install/config-cloudfront.rst

```diff
@@ -3,58 +3,69 @@
 Configuring CloudFront to host Mattermost static assets
 =======================================================
 
-Configuring CloudFront to host Mattermost's static assets allows for improved caching performance and shorter load times for those members of your team geographicly distributed throughout the world. 
+Configuring CloudFront to host Mattermost's static assets allows for improved caching performance and shorter load times for those
+members of your team geographicly distributed throughout the world.
 
 1. Create an S3 bucket using your desired domain. In our example it will be ``mattermost.example.com``.
 2. Enable static hosting for your S3 bucket.
-3. If your SiteURL is configured with a subpath (such as https://example.com/company/mattermost), your static assets must be rewritten before uploading. You can use the command below to rewrite the assets for the given subpath:
```

Comment:
Did you intentionally break long lines into separate shorter lines?

Reply:
I did in my editor, for ease of editing, but the output doesn't get affected by the line breaks.